crypto/rsa/rsa_backend.c

```diff
@@ -7,10 +7,18 @@
  * https://www.openssl.org/source/license.html
  */
 
+#include <string.h>
 #include <openssl/core_names.h>
 #include <openssl/params.h>
+#include <openssl/evp.h>
+#include "internal/sizes.h"
+#include "internal/param_build_set.h"
 #include "crypto/rsa.h"
 
+#ifdef _WIN32
+# define strcasecmp _stricmp
```

Comment:
Why are you doing this when it is defined somewhere already?

Reply:
e_os.h is a mess, it seems unnecessary to include it for just this.

Reply:
bleah, that's a bad reason.  fix e_os.h if that's the core problem, don't monkey-patch around it.

Reply:
Sure, it's in my backlog, after the stuff that I need to get ready before beta

Reply:
But the point is that this simple three-line hack adds to technical debt.  You know have to find all the cases where the define is used.

And especially, if you intend to work on e_os.h then this is misguided.

Yes, this is a lot of commentary for a simple hack.  That's what makes us programmers ;)